frontend/app/route-helpers/status-route-helpers.server.ts

```diff
@@ -0,0 +1,156 @@
+import type { Session } from '@remix-run/node';
+import { redirectDocument } from '@remix-run/node';
+import type { Params } from '@remix-run/react';
+
+import { UTCDate } from '@date-fns/utc';
+import { differenceInMinutes } from 'date-fns';
+import { omit } from 'moderndash';
+import { z } from 'zod';
+
+import type { ContextualAlertType } from '~/utils/application-code-utils.server';
+import { getLocaleFromParams } from '~/utils/locale-utils.server';
+import { getLogger } from '~/utils/logging.server';
+import { getCdcpWebsiteApplyUrl } from '~/utils/url-utils.server';
+
+const log = getLogger('apply-route-helpers.server');
+
+export interface StatusState {
+  readonly lastUpdatedOn: string;
+  readonly id: string;
+  readonly statusCheckResult?: {
+    alertType: ContextualAlertType;
+    name?: string;
+    id?: string;
+    statusId?: string | null;
+  };
+}
+
+/**
+ * Schema for validating UUID.
+ */
+const idSchema = z.string().uuid();
+
+/**
+ * Gets the session name.
+ * @param id - The ID.
+ * @returns The session name.
+ */
+function getSessionName(id: string) {
+  return `status-flow-${idSchema.parse(id)}`;
+}
+
+interface LoadStateArgs {
+  params: Params;
+  session: Session;
+}
+
+/**
+ * Loads state.
+ * @param args - The arguments.
+ * @returns The loaded state.
+ */
+export function loadStatusState({ params, session }: LoadStateArgs) {
+  const locale = getLocaleFromParams(params);
+  const cdcpWebsiteApplyUrl = getCdcpWebsiteApplyUrl(locale);
```

Comment:
I feel we should redirect to the status page rather than the apply page of the CDCP website.

```suggestion
  const cdcpWebsiteStatusUrl = getCdcpWebsiteStatusUrl(locale);
```

Reply:
Changed link